testcases/OpTestSecvarctl.py

```diff
@@ -0,0 +1,92 @@
+import unittest
+import os
+import time
+
+import OpTestConfiguration
+import OpTestLogger
+from common.OpTestUtil import OpTestUtil
+
+log = OpTestLogger.optest_logger_glob.get_logger(__name__)
+
+
+class SecvarctlTest(unittest.TestCase):
+    def setUp(self):
+        self.conf = OpTestConfiguration.conf
+        self.util = OpTestUtil(OpTestConfiguration.conf)
+        self.cv_HOST = self.conf.host()
+        self.distro_name = self.util.distro_name()
+        self.cv_SYSTEM = self.conf.system()
+        self.connection = self.cv_SYSTEM.cv_HOST.get_ssh_connection()
+        self.secvar_repo = self.conf.args.git_repo
+        self.branch = self.conf.args.git_branch
+        self.host_cmd_timeout = self.conf.args.host_cmd_timeout
+        self.home = self.conf.args.git_home
+        dep_packages = ['gcc', 'make', 'cmake', 'git', 'libopenssl-devel']
+        if self.distro_name == 'rhel':
+            self.installer = "dnf install"
+        elif self.distro_name == 'sles':
+            self.installer = "zypper install"
+        for pkg in dep_packages:
+            if self.distro_name == 'rhel':
+                self.cv_HOST.host_run_command(f"{self.installer} {pkg} -y")
+            elif self.distro_name == 'sles':
+                self.cv_HOST.host_run_command(f"{self.installer} -y {pkg}")
```

Comment:
please move this logic in utils . and we can drive that from test like just passing package name